Makefile

```diff
@@ -21,3 +21,13 @@ helm_dep_up:
 
 .PHONY: update_dependencies
 update_dependencies: setup_helm_repos helm_dep_up update_worker_version
+
+.PHONY: get_release_notes
+get_release_notes: setup_helm_repos helm_dep_up
+	git clone git@github.com:codacy/release-notes-tools.git
```

Comment:
Because generating an installable package out of release-notes-tools is out of the immediate scope of this task, this is how we install it.